src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1481,9 +1500,21 @@ void Convolution::prepareParams() {
 
     if (execPtr) {
         primArgs[DNNL_ARG_SRC] = srcMemPtr->GetPrimitive();
-        primArgs[DNNL_ARG_WEIGHTS] = wghMemPtr->GetPrimitive();
         primArgs[DNNL_ARG_DST] = dstMemPtr->GetPrimitive();
 
+        if (key.constWeight) {
+            // const weight preparation/reordering needs to be done once at next execution
+            // when the input weight data is guaranteed to be ready (considering possible const-folding
+            // subgraphs inserted between constant weight node and conv)
+            auto it = primArgs.find(DNNL_ARG_WEIGHTS);
+            if (it == primArgs.end() || prevExecPtr->getWeightDesc() != execPtr->getWeightDesc()) {
```

Comment:
It does make sense to add `prevExecPtr` null check to prevent nullptr dereferencing.

Reply:
OK, will add `|| !prevExecPtr ||` into the condition so when there is no valid `prevExecPtr` (which is rare) we will also reorder the constant weight.